projects/plugins/jetpack/modules/theme-tools/site-logo.php

```diff
@@ -22,13 +22,18 @@
  * Activate the Site Logo plugin.
  *
  * @uses current_theme_supports()
- * @since 3.2
+ * @since 3.2.0
+ * @since 9.9.0 Uses Core site_logo option format universally.
  */
 function site_logo_init() {
-	// For transferring existing site logo from Jetpack -> Core
-	if ( current_theme_supports( 'custom-logo' ) && ! get_theme_mod( 'custom_logo' ) && $jp_logo = get_option( 'site_logo' ) ) {
-		set_theme_mod( 'custom_logo', $jp_logo['id'] );
-		delete_option( 'site_logo' );
+	// For transferring existing site logo from Jetpack -> Core.
+	if ( current_theme_supports( 'custom-logo' ) && ! get_theme_mod( 'custom_logo' ) && get_option( 'site_logo' ) ) {
+		$jp_logo = get_option( 'site_logo' );
```

Comment:
Would it make sense to move the `get_option` here?

```suggestion
	$jp_logo = get_option( 'site_logo' );
	if ( current_theme_supports( 'custom-logo' ) && ! get_theme_mod( 'custom_logo' ) && $jp_logo ) {
		
```

Reply:
I thought about that, but didn't because it would mean a db call that might not be needed if `current_theme_support` was false.